Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Instant_Message.yml

```diff
@@ -0,0 +1,338 @@
+      - "5"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      gridfield:
+        simple: asmnotification
+      keys:
+        simple: type,value,url,timestamp
+      val1:
+        simple: Slack
+      val2:
+        simple: ${Slack.Thread.ID}
+      val3:
+        simple: n/a
+      val4:
+        simple: TIMESTAMP
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: |-
+        Automation used to more easily populate a grid field. This is necessary when you want to assign certain values as static or if you have context paths that you will assign to different values as well. For example:
+        `!GridFieldSetup keys=ip,src val1=${AWS.EC2.Instances.NetworkInterfaces.PrivateIpAddress} val2="AWS" gridfiled="gridfield"`
+      id: e529a3dc-af23-41e5-8445-64473ce4b094
+      iscommand: false
+      name: Set notification grid field
+      script: GridFieldSetup
+      type: regular
+      version: -1
+    taskid: e529a3dc-af23-41e5-8445-64473ce4b094
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 360
+        }
+      }
+  "10":
+    continueonerrortype: ""
+    id: "10"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "8"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      blocks:
+        simple: |-
+          [
+              {
+                  "type": "rich_text",
+                  "elements": [
+                      {
+                          "type": "rich_text_section",
+                          "elements": [
+                              {
+                                  "type": "link",
+                                  "text": "Cortex ASM Alert: ${alert.name}",
+                                  "url": "${AlertURL}"
+                              }
+                          ]
+                      }
+                  ]
+              },
+              {
+                  "type": "divider"
+              },
+              {
+                  "type": "rich_text",
+                  "elements": [
+                      {
+                          "type": "rich_text_section",
+                          "elements": [
+                              {
+                                  "type": "text",
+                                  "text": "${FormattedBody}"
+                              }
+                          ]
+                      }
+                  ]
+              },
+              {
+                  "type": "divider"
+              }
+          ]
+      channel_id:
+        simple: ${inputs.InstantMessageChannel}
+    separatecontext: false
+    skipunavailable: true
+    task:
+      brand: SlackV3
+      description: Sends a message to a user, group, or channel.
+      id: dfb76202-de06-43c1-82b4-7fe51c9828d5
+      iscommand: true
+      name: Send Slack message
```

Comment:
Before sending the Slack message we should check that slack is enabled, unless you think the check in RPR is enough.

Reply:
yes, we are effectively checking slack is enabled in RPR (just like for Jira and SNOW)